app/components/Views/SendFlow/SendTo/index.js

```diff
@@ -358,8 +358,35 @@ class SendFlow extends PureComponent {
     });
     this.toggleFromAccountModal();
   };
+  /**
+   * This returns the address name from the address book or user accounts if the selectedAddress exist there
+   * @param {*} toSelectedAddress - Represents the value of what user writes on toAddress input
+   * @returns
+   */
+  getAddressNameFromBookOrIdentities = (toSelectedAddress) => {
+    if (!toSelectedAddress) return;
+
+    const { addressBook, network, identities } = this.props;
+    const networkAddressBook = addressBook[network] || {};
+
+    const checksummedToSelectedAddress = toChecksumAddress(toSelectedAddress);
```

Comment:
Improve variable name
```suggestion
    const checksummedAddress = toChecksumAddress(toSelectedAddress);
```

Reply:
Thank you, much better!